tests/test_core.py

```diff
@@ -902,7 +902,7 @@ def test_config_defaults():
     assert config.time_zone == "UTC"
     assert config.internal_url is None
     assert config.external_url is None
-    assert config.config_source == "default"
+    assert config.config_source is ha.ConfigSource.DEFAULT
```

Comment:
Is there a reason for importing these differently to the rest?

It works, I just wondered if there was a reason.

Reply:
This ensures it is a ConfigSource, and not string matched.

Reply:
@frenck Sure, i just meant why 
`ha.ConfigSource.DEFAULT`
rather than 
`ConfigSource.DEFAULT`
which is how it looks in the rest of the PR.

Reply:
This PR yes, this specific file: no.
It doesn't matter really.